Riot/Modules/Room/Location/RoomTimelineLocationView.swift

```diff
@@ -19,26 +19,60 @@ import Reusable
 import Mapbox
 import SwiftUI
 
-struct LiveLocationParameter {
-    let bannerImage: UIImage
-    let bannerTitle: String
-    let timer: String?
-    let shouldShowStopButton: Bool
-    let isLive: Bool
+protocol RoomTimelineLocationViewDelegate: AnyObject {
+    func didTapStopButton()
+    func didTapRetryButton()
```

Comment:
In general we are referencing the delegate instance:
```
    func roomTimelineLocationViewDidTapStopButton(_ roomTimelineLocationView: RoomTimelineLocationView)
    func roomTimelineLocationViewDidTapRetryButton(_ roomTimelineLocationView: RoomTimelineLocationView)
```
Note: You can also use closures if want instead of delegation